src/custom/components/WalletModal/WalletModalMod.tsx

```diff
@@ -24,12 +24,12 @@ import { FortmaticOption } from 'components/WalletModal/FortmaticOption'
 import { InjectedOption, InstallMetaMaskOption, MetaMaskOption } from 'components/WalletModal/InjectedOption'
 import PendingView from 'components/WalletModal/PendingView'
 import { WalletConnectOption } from 'components/WalletModal/WalletConnectOption'
-import { WalletConnect } from '@web3-react/walletconnect'
 
 // MOD imports
 import ModalMod from '@src/components/Modal'
 import { changeWalletAnalytics } from 'components/analytics'
 import usePrevious from 'hooks/usePrevious'
+import type { WalletConnect } from '@web3-react/walletconnect'
```

Comment:
Did the WC version change? Otherwise, how did it work to import the same name but now as a explicit type?

Reply:
Before we "physically" imported code from `@web3-react/walletconnect` and we got it in the bundle.
Now we don't have it in the bundle, just use it on the compilation level (typescript).